samples/host_verify/host.c

```diff
@@ -189,15 +189,15 @@ oe_result_t verify_evidence(
     return result;
 }
 
-oe_result_t sgx_enclave_claims_verifier(
+oe_result_t custom_enclave_claims_verifier(
```

Comment:
The claims being verified in the function are common claims that should be supported by all TEEs; not SGX claims or custom claims. I'd suggest the function to be named enclave_claims_verifier().